examples/tests/Earthfile

```
@@ -714,6 +715,42 @@ true-false-flag-invalid:
     DO +RUN_EARTHLY --earthfile=true-false-flag-invalid.earth --should_fail=true --target=+run-maybe --post_command="2>output3.txt"
     RUN cat output3.txt | grep 'invalid argument for flag .*--no-cache.*expected bool'
 
+sequential-locally-test:
+    DO +RUN_EARTHLY --earthfile=sequential-locally.earth --use_tmpfs=false --target="+run-lots" --extra_args="--no-cache" --post_command="2>output.txt"
```

`--no-cache` isn't really needed given that these are LOCALLY targets, right?

```suggestion
    DO +RUN_EARTHLY --earthfile=sequential-locally.earth --use_tmpfs=false --target="+run-lots" --post_command="2>output.txt"
```

---

good point, it won't have any effect as LOCALLY commands are never cached.